README.md

```diff
@@ -673,7 +672,22 @@ We alway welcome your contributions :clap:
 
 Bugs, feature requests and comments are more than welcome in the [issues].
 
-### Feedback
+## Release Process
+
+There is a `release` target within the Makefile that wraps up the steps to release a new version.
+
+> NOTE: Pass the `VERSION` variable when running the command to properly set the tag version for the release.
+
+```
```

Comment:
Nit: and to appease lint warning:

```suggestion
```bash
$ VERSION=vX.Y.Z make release
# EXAMPLE:
$ VERSION=v0.11.3 make release
```

Reply:
Updated, thanks!